includes/class-wc-install.php

```diff
@@ -670,6 +671,16 @@ private static function get_schema() {
   PRIMARY KEY (log_id),
   KEY level (level)
 ) $collate;
+CREATE TABLE {$wpdb->prefix}woocommerce_order_product_lookup (
+  lookup_id BIGINT UNSIGNED NOT NULL auto_increment,
+  user_email varchar(200) NOT NULL,
```

Comment:
User email in users table is 100 chars. But post meta where billing is stored is unlimited. Whats best here? Do we need more validation for billing email?